lib/sensors/accelerometer.cpp

```diff
@@ -0,0 +1,145 @@
+#include "accelerometer.hpp"
+
+namespace hyped::sensors {
+
+Accelerometer::Accelerometer(std::uint8_t channel, io::HardwareI2c &i2c, core::ILogger &logger)
```

Comment:
`const` for channel

Reply:
also i believe args are ordered from most to least "standard" ie logger, i2c, channel